dictionary/index.js

```
@@ -21,20 +21,16 @@ const locales = {
 const dictionary = new Proxy(locales, {
     get(target, prop) {
         if (prop === 'set') {
-            return (provider) => {
-                if (!Validation.isServiceProvider(provider)) {
-                    throw new Error(`Invalid service provider name: "${provider}"`);
-                }
-                
-                const filename = provider + '.local.js';
+            return (name) => {
+                const filename = name + '.local.js';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)